docs/sources/auth/generic-oauth.md

```diff
@@ -13,7 +13,7 @@ weight = 3
 # Generic OAuth Authentication
 
 You can configure many different OAuth2 authentication services with Grafana using the generic OAuth2 feature. Below you
-can find examples using Okta, BitBucket, OneLogin and Azure.
+can find examples using BitBucket, OneLogin, Auth0 and Centrify. [Okta]({{< relref "okta.md" >}}) and [Azure AD]({{< relref "azuread.md" >}}) have moved to separate pages.
```

Comment:
```suggestion
can find examples using BitBucket, OneLogin, Auth0, and Centrify. [Okta]({{< relref "okta.md" >}}) and [Azure AD]({{< relref "azuread.md" >}}) have moved to separate pages.
```

Reply:
Make each option a bullet in an unordered list. The two bullets on other pages are links, the rest just say "see below" or link to the appropriate subheading.

Reply:
Thanks for the suggestion! I agree it's better with a bullet list. Hope the syntax for subheading links is correct. If not I'll do a commit amend. Don't know how to build the docs locally to test the result.

Reply:
Instructions are in the README. https://github.com/grafana/grafana/tree/master/docs#building-the-docs-locally

Reply:
Should have said this as well: please make the list alphabetical or use some other logical sorting method.

Reply:
Should I also move the samples around, so they are in the same order as the list? Now the bullet list is in the same order as the samples below.

Reply:
Yes, definitely.

Reply:
If you want to upgrade your technical writing skills for free, I highly recommend reading through Google's Technical Writing for Engineers course. https://developers.google.com/tech-writing/overview

Reply:
Thank you for that recommendation. I'm actually thinking about starting a tech blog, so it's a great time to take that course.

Ordered list is pushed now and building.